ts/Series/PieSeries.ts

```diff
@@ -864,6 +862,16 @@ BaseSeries.seriesType<typeof Highcharts.PieSeries>(
          */
         generatePoints: function (this: Highcharts.PieSeries): void {
             LineSeries.prototype.generatePoints.call(this);
+            const data = this.data;
```

Comment:
Why do we need this new block? The original fiddle seems to work without it...

Reply:
The fix replaces negative values with null on the processed data. Data labels require null values on point instance to align correctly to their related element.

This is the incorrect rendering without the block:
![highcharts-item-chart](https://user-images.githubusercontent.com/14174575/96227977-36457f00-0f95-11eb-93b5-c63df2f71191.png)

This is the correct rendering with the block:
![highcharts-item-chart (1)](https://user-images.githubusercontent.com/14174575/96227968-334a8e80-0f95-11eb-904a-1ed3121e6dc4.png)

Reply:
I see! I moved this check to the translate function where the angles are calculated, I think that's sufficient. Commit coming up.